src/main/java/org/embulk/output/kintone/KintonePageOutput.java

```diff
@@ -104,133 +108,171 @@ public void connect(final PluginTask task) {
     }
   }
 
+  private void update(ArrayList<RecordForUpdate> records) {
+    execute(
+        client -> {
+          client.record().updateRecords(task.getAppId(), records);
+        });
+  }
+
+  private void insert(ArrayList<Record> records) {
+    execute(
+        client -> {
+          client.record().addRecords(task.getAppId(), records);
+        });
+  }
+
   private void execute(Consumer<KintoneClient> operation) {
     connect(task);
-    if (this.client != null) {
-      operation.accept(this.client);
-    } else {
+    if (this.client == null) {
       throw new RuntimeException("Failed to connect to kintone.");
     }
+    try {
+      retryExecutor()
+          .withRetryLimit(10)
+          .withInitialRetryWait(1000)
+          .withMaxRetryWait(60 * 1000)
+          .runInterruptible(
+              new Retryable<Void>() {
+
+                @Override
+                public Void call() throws Exception {
+                  operation.accept(client);
+                  return null;
+                }
+
+                @Override
+                public boolean isRetryableException(Exception e) {
+                  if (!(e instanceof KintoneApiRuntimeException)) {
+                    return false;
+                  }
+
+                  int code = ((KintoneApiRuntimeException) e).getStatusCode();
```

Comment:
fixed: de71e7c

Reply:
Unfortunately, there are no list of error codes.
These errors I have encountered.
https://github.com/trocco-io/embulk-output-kintone/blob/1fd3cf43d77823b9cb29bd714157550aa93d23f5/src/main/java/org/embulk/output/kintone/KintonePageOutput.java#L41-L43

Reply:
As you say, there seems to be no documentation of error codes.
If so, to make it more resistant to change, 
it might be better to allow retryable http status codes and  api error codes to be set from the config, 
but this seems too complicated, so the current code is sufficient.